model/command_response.go

```diff
@@ -18,17 +18,18 @@ const (
 )
 
 type CommandResponse struct {
-	ResponseType   string             `json:"response_type"`
-	Text           string             `json:"text"`
-	Username       string             `json:"username"`
-	ChannelId      string             `json:"channel_id"`
-	IconURL        string             `json:"icon_url"`
-	Type           string             `json:"type"`
-	Props          StringInterface    `json:"props"`
-	GotoLocation   string             `json:"goto_location"`
-	TriggerId      string             `json:"trigger_id"`
-	Attachments    []*SlackAttachment `json:"attachments"`
-	ExtraResponses []*CommandResponse `json:"extra_responses"`
+	ResponseType     string             `json:"response_type"`
+	Text             string             `json:"text"`
+	Username         string             `json:"username"`
+	ChannelId        string             `json:"channel_id"`
+	IconURL          string             `json:"icon_url"`
+	Type             string             `json:"type"`
+	Props            StringInterface    `json:"props"`
+	GotoLocation     string             `json:"goto_location"`
+	TriggerId        string             `json:"trigger_id"`
+	SkipSlackParsing bool               `json:"skip_slack_parsing"`
```

Comment:
I know this code piece is barely document, but should you please take the first step and add a godoc comment to `SkipSlackParsing`?